jmix-translations/content/io/jmix/securityflowui/messages.properties

```diff
@@ -192,9 +192,22 @@ io.jmix.securityflowui.view.changepassword/changePasswordView.passwordChanged=Pa
 io.jmix.securityflowui.view.changepassword/changePasswordView.emptyUsernameMessage=Dialog cannot be opened without username
 io.jmix.securityflowui.view.changepassword/changePasswordView.title=Change password for [%s]
 
-io.jmix.securityflowui.view.resetpassword/resetPasswordView.resetPasswordsTitle = System will generate new password to selected users
-io.jmix.securityflowui.view.resetpassword/resetPasswordView.printPasswords=Generated passwords will be shown below after clicking the "Generate and save" button
-io.jmix.securityflowui.view.resetpassword/resetPasswordView.generateSave=Generate and save
+io.jmix.securityflowui.view.resetpassword/resetPasswordView.resetPasswordsTitle = Generate new passwords for selected users
+io.jmix.securityflowui.view.resetpassword/resetPasswordView.resetSinglePasswordTitle = Generate a new password for [%s]
+io.jmix.securityflowui.view.resetpassword/resetPasswordView.resetPasswordLabel.text = Automatically generate a password
```

Comment:
Let's extend description, e.g.:
When you proceed, new password(s) will be generated and assigned to selected user(s)